lib/kubeclient/common.rb

```diff
@@ -254,6 +241,29 @@ def proxy_url(kind, name, port, namespace = '')
       end
     end
 
+    def get_pod_log(pod_name, namespace = nil, container: nil, previous: false)
+      params = { params: {} }
+      params[:params][:previous] = true if previous
+      params[:params][:container] = container if container
+
+      ns = build_namespace_prefix(namespace || 'default')
```

Comment:
Okay, good point.  I can remove that.

Also, now that PR #105 is merged, I'll change `namespace` to be part of the options hash, rather than a regular parameter.